nginx/custom-nginx.conf

```
@@ -0,0 +1,12 @@
+proxy_connect_timeout       600;
+proxy_send_timeout          600;
+proxy_read_timeout          600;
+send_timeout                600;
+
+# Also increase (header) buffer size
+proxy_buffer_size   128k;
+proxy_buffers   4 256k;
+proxy_busy_buffers_size   256k;
+
+# Fix issues with large json responses
+proxy_max_temp_file_size 0;
```

No newline at end of file
```suggestion
proxy_max_temp_file_size 0;

```